optaplanner-examples/src/main/java/org/optaplanner/examples/cheaptime/solver/drools/Period.java

```diff
@@ -0,0 +1,52 @@
+package org.optaplanner.examples.cheaptime.solver.drools;
+
+import org.apache.commons.lang.builder.CompareToBuilder;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+public class Period {
```

Comment:
The class name Period creates confusion: every where else vars names period are of the type int.
I am going to rename this class to PeriodWrapper to avoid ambiguity.